Anytype/Sources/PresentationLayer/Common/SwiftUI/StatusBarStyle/StatusBarStyleModifier.swift

```diff
@@ -26,6 +26,7 @@ struct StatusBarStyleModifier: ViewModifier {
     
     let style: AnytypeStatusBarStyle
     
+    @available(*, deprecated) // Workround for disable deprecated API
```

Comment:
Workround for disable deprecated API **warning**

Reply:
fixed